lib/src/main/java/com/evervault/services/HttpHandler.java

```diff
@@ -160,6 +164,64 @@ public <T> T decrypt(String url, Object data, Class<T> valueType) throws HttpFai
         return new Gson().fromJson(response.body(), valueType);
     }
 
+    @Override
+    public TokenResult createClientSideDecryptToken(String url, String action, Object data, Instant expiry) throws HttpFailureException, IOException, InterruptedException {
```

Comment:
nit: Should this name be more generic since it takes in actual as a parameter, so I'd imagine we'll be using it for function tokens in the future.

Reply:
Similar for `CreateDecryptTokenPayload`, perhaps it should just be `TokenPayload`